client/src/App.jsx

```diff
@@ -1,17 +1,34 @@
-import React from 'react';
+import React, { Component, Fragment } from 'react';
 import { BrowserRouter, Route, Switch } from 'react-router-dom';
 import Login from './components/Login';
 import './App.css';
 import NewTicket from './components/NewTicket';
 import MainSidebar from './components/MainSidebar';
+import TicketsSidebar from './components/TicketsSidebar';
 
-export default () => (
-  <BrowserRouter>
-    <Switch>
-      <Route exact path="/" />
-      <Route exact path="/new-ticket" component={NewTicket} />
-      <Route exact path="/login" component={Login} />
-      <Route exact path="/main-sidebar" component={() => <MainSidebar selected="tickets" />} />
-    </Switch>
-  </BrowserRouter>
-);
+class App extends Component {
+  render() {
+    return (
+      <BrowserRouter>
+        <Switch>
+          <Route
+            exact
+            path="/"
+            render={
```

Comment:
these should be wrapped inside the main page component.
```suggestion
            component={Home} // or whatever you would like to name the home page component.
```